.github/PULL_REQUEST_TEMPLATE/analyst_experience_team.md

```diff
@@ -0,0 +1,48 @@
+## Summary
+
+Summarize your PR. If it involves visual changes include a screenshot or gif.
+
+- **Related issue:** < LINK_TO_RELATED_ISSUE >
+- **Feature flag:** < FEATURE_FLAG_HERE >
+- **Docs ticket:** < LINK_TO_DOCS_TICKET_HERE >
+- **Schema changes:** < LINK_SCHEMA_CHANGE_TICKET_HERE >
+
+### How to test
+
+Provide steps for reviewers to know how to test your PR. Assume that a reviewer may have little to no context when determining how detailed this section should be.
+
+### Release Notes
+
+The text that appears in the Release Notes is pulled directly from your PR title, or a single paragraph of text that you specify in the PR description. See [Release Notes Guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process).
+
+### Definition of done
+
+Check the PR satisfies following conditions.
+
+- [ ] Functional changes are hidden behind a feature flag. If not hidden, the PR explains why these changes are being implemented in a long-living feature branch. - [Feature flag guidelines](https://docs.elastic.dev/security-solution/process/detections/pull-requests#feature-toggling-for-everything)
+- [ ] Functional changes are covered with a test plan and automated tests.
+- [ ] PR is tested within MKI environment. MKI environment behavor differs and is not checked during the PR CI process. See guidelines for testing on MKI - [Manually testing PRs on MKI environments](https://docs.elastic.dev/security-solution/teams/analyst-experience/manual-testing/serverless/1.testing-PRs-on-MKI.mdx)
+- [ ] Stability of new and changed tests is verified using the [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) in both ESS and Serverless. By default, use 200 runs for ESS and 200 runs for Serverless.
```

Comment:
It would be great if for the stability section we have links to the [main causes of flakiness](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/flaky-tests/5.main-causes) so people know how to better code reliable tests, as well as the [prevention](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/flaky-tests/6.preventing-flakiness) section :)